cmd/osv-scanner/flaghelpers.go

```diff
@@ -0,0 +1,43 @@
+package main
```

Comment:
nit: helpers is generlaly not a good name. Can we rename this to callanalysis_state.go instead?